### DISCUSSION TITLE
ShokaX无法正常加载的解决方案

### DISCUSSION THREAD

**Discussion by zkz098** [theme-shoka-x/hexo-theme-shokaX#92] [Announcements]
近期收到了一些ShokaX无法正常加载的反馈（也包括我看到的一部分），这部分无法加载多是因为配置不当造成的，可分为如下几种:
1. `rocket-loader`配置不当(0.2.6已修复)
    ShokaX page ts和cloudflare CDN的`rocket-loader`存在冲突，开启`rocket-loader`后加载会被阻断，表现为卡loading并且无控制台报错
    可更新到0.2.6+解决
2. CDN加载被阻断
    使用了访问不畅的CDN导致依赖项被阻断，这也会导致无法正常加载，表现为卡loading，强行进入后页面无内容（仅有骨架），有控制台报错，解决方案是修改advVendors的加载源为访问正常的CDN
3. vendors未更新
    在引入了ShokaX-pjax和ShokaX-anime后，未更新advVendors就升级了版本（复制后的设置覆盖了主题默认设置）
    解决方案是直接删除`_config.shokaX.yml`中的advVendors和vendors

**Comment by Nightingale0504** [comment]
> 近期收到了一些ShokaX无法正常加载的反馈（也包括我看到的一部分），这部分无法加载多是因为配置不当造成的，可分为如下几种:
> 1. `rocket-loader`配置不当(0.2.6已修复)
>     ShokaX page ts和cloudflare CDN的`rocket-loader`存在冲突，开启`rocket-loader`后加载会被阻断，表现为卡loading并且无控制台报错
>     可更新到0.2.6+解决
> 2. CDN加载被阻断
>     使用了访问不畅的CDN导致依赖项被阻断，这也会导致无法正常加载，表现为卡loading，强行进入后页面无内容（仅有骨架），有控制台报错，解决方案是修改advVendors的加载源为访问正常的CDN
> 3. vendors未更新
>     在引入了ShokaX-pjax和ShokaX-anime后，未更新advVendors就升级了版本（复制后的设置覆盖了主题默认设置）
>     解决方案是直接删除`_config.shokaX.yml`中的advVendors和vendors

如果我并未配置advVendors和vendors但是访问站点时会出现情况2描述现象，我该如何解决问题

**Reply by zkz098** [reply]
大部分js运行错误都是2的现象，浏览器F12控制台报错发一下

**Reply by Nightingale0504** [reply]
> 大部分js运行错误都是2的现象，浏览器F12控制台报错发一下

昨天好了，应该是cdn没加载完的过，抱歉麻烦您了
